.eslintrc.js

```diff
@@ -3,20 +3,29 @@ module.exports = {
     browser: true,
     es2021: true,
   },
-  extends: ['eslint:recommended', 'plugin:react/recommended'],
+  extends: [
+    'eslint:recommended',
+    'plugin:import/recommended',
+    'plugin:jsx-a11y/recommended',
+    'plugin:react/recommended',
+    'plugin:react-hooks/recommended',
+  ],
   parserOptions: {
     ecmaFeatures: {
       jsx: true,
     },
     ecmaVersion: 13,
     sourceType: 'module',
   },
-  plugins: ['react'],
   rules: {
     curly: 'error',
     'default-case': 'error',
     'dot-notation': 'error',
     eqeqeq: 'error',
+    // TODO: Turn import/no-unresolved back on once webpack is configured.
+    // See https://github.com/import-js/eslint-plugin-import/blob/main/docs/rules/no-unresolved.md
+    // and https://github.com/import-js/eslint-plugin-import/blob/main/README.md#resolvers for details.
+    'import/no-unresolved': 'off',
```

Comment:
this errors right now because we use relative imports within packages like this:

https://github.com/code-dot-org/dsco_/blob/10d0c42fced221c4ce17d1b48b317a1aa18557dd/packages/link/src/index.js#L1

we should be able to turn this back on once webpack is configured and we add the [eslint-import-resolver-webpack plugin](https://www.npmjs.com/package/eslint-import-resolver-webpack)